src/animation/AnimationMixer.js

```diff
@@ -199,42 +315,139 @@ THREE.AnimationMixer.prototype = {
 
 	update: function( deltaTime ) {
 
-		var mixerDeltaTime = deltaTime * this.timeScale;
-		this.time += mixerDeltaTime;
+		var actions = this._actions,
+			mixerDeltaTime = deltaTime * this.timeScale;
 
-		for ( var i = 0; i < this.actions.length; i ++ ) {
+		var time = this.time += mixerDeltaTime;
+		var accuIndex = this.accuIndex ^= 1;
```

Comment:
The concept is called "double buffering": Prominently known from graphics, actually: You display one screenful of video memory while rendering to the other. When done, the framebuffer base address is switched and the rendered frame is displayed while rendering proceeds on the previously displayed memory. This way, no copy is needed (that's also why the GL ES specification warns about asking low-end hardware to keep the framebuffer contents).

Same thing happens here: By using the accumulation regions in an interleaved fashion, I can compare the "frames" without having to copy the value of the previous frame (which can then take an early exit for vectored data).

There's a shorter explanation in the comment about the buffer layout in the constructor.